includes/blocks/class-sensei-block-contact-teacher.php

```diff
@@ -104,15 +103,24 @@ private function teacher_contact_form( $post ) {
 	/**
 	 * Add attributes to the block's <a> tag.
 	 *
-	 * @param string $content Block HTML.
-	 * @param string $href    Link URL.
+	 * @param string $content    Block HTML.
+	 * @param string $href       Link URL.
+	 * @param array  $attributes Block attributes.
 	 *
 	 * @return string Block HTML with additional href attribute.
 	 */
-	private function add_button_attributes( $content, $href ) {
+	private function add_button_attributes( $content, $href, $attributes = [] ) {
+		$class_name = Sensei_Block_Helpers::block_class_with_default_style( $attributes, [], 'outline' );
+
 		return preg_replace(
-			'/<a(.*)class="(.*)"(.*)>(.+)<\/a>/',
-			'<a href="' . esc_url( $href ) . '#private_message" class="sensei-collapsible__toggle $2" $1 $3>$4</a>',
+			'/<div(.*)class="(.*)"(.*)>(.*)<a(.*)class="(.*)"(.*)>(.+)<\/a>(.*)<\/div>/',
```

Comment:
I wish there was a better way to handle this than using regexps. Can we at least break it into different ones to make it simpler?

Reply:
I agree! Maybe in the future, we could explore a little more this solution to think in a way more simple to do that. For now, I just broke the replaces as suggested. https://github.com/Automattic/sensei/pull/3787/commits/9e3c7b830996659484a0358b0ac4ff0217ea1062

Reply:
I thought a little more about this PR solution, and I did a refactor that made all the solution more simple: https://github.com/Automattic/sensei/pull/3787/commits/1d834adc427f24db999c7f9e77f976e0bcd50c23

Basically, instead of adding the style class through the server, I added it in the block save.